packages/@aws-cdk/toolkit-lib/lib/toolkit/toolkit.ts

```diff
@@ -246,8 +246,6 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
 
     await migrator.tryMigrateResources(stackCollection, options);
 
-    const requireApproval = options.requireApproval ?? RequireApproval.NEVER;
```

Comment:
this breaks the behavior of the `requireApproval` option but i don't have a good way or hooking it up to the `requireApproval` governed by the `CliIoHost` -- `toolkit.ts` only has a concept of an `IIoHost`